strings/tips.txt

```diff
@@ -242,9 +242,6 @@ As a Morph, you can talk while disguised, but your words have a chance of being
 As a Drone, you can ping other drones to alert them of areas in the station in need of repair.
 As a Ghost, you can see the inside of a container on the ground by clicking on it.
 As a Ghost, you can double click on just about anything to follow it. Or just warp around!
-As a Devil, you gain power for every three souls you control, however you also become more obvious.
-As a Devil, as long as you control at least one other soul, you will automatically resurrect, as long as a banishment ritual is not performed.
-At which time a Devil's nameth is spake on the tongue of man, the Devil may appeareth.
```

Comment:
Maybe move this one to sillytips?

Reply:
I don't think it's silly enough for that tbh

Reply:
its funny keep it in